drivers/scheduler/k8s/k8s.go

```diff
@@ -538,6 +553,23 @@ func (k *k8s) substituteNamespaceInPVC(pvc *v1.PersistentVolumeClaim, ns string)
 	}
 }
 
+func (k *k8s) addSecurityAnnotation(spec interface{}, configMap *v1.ConfigMap) {
+	logrus.Infof("Config Map details:\n %v:", configMap.Data)
+	if obj, ok := spec.(*v1.PersistentVolumeClaim); ok {
+		if obj.Annotations == nil {
+			obj.Annotations = map[string]string{}
+		}
+		obj.Annotations[secretName] = configMap.Data["secret_name"]
+		obj.Annotations[secretNamespace] = configMap.Data["secret_namespace"]
+	} else if obj, ok := spec.(*snap_v1.VolumeSnapshot); ok {
+		if obj.Metadata.Annotations == nil {
+			obj.Metadata.Annotations = map[string]string{}
+		}
+		obj.Metadata.Annotations[secretName] = configMap.Data["secret_name"]
+		obj.Metadata.Annotations[secretNamespace] = configMap.Data["secret_namespace"]
+	}
```

Comment:
Use generic objects to update annotations. Do this for ApplicationClone, ApplicationBackup, Application restore, MIgration and VolumeSnapshotRestore.

Reply:
Discussed with @disrani-px we can skip this for now.